Hello!

Sorry, I got a little impatient! 😅

When I use your derivation, I'm getting this error:
```
go test proxy running at GOPROXY=http://127.0.0.1:44259/mod
--- FAIL: TestScript (0.11s)
    --- FAIL: TestScript/mod_find (0.32s)
        script_test.go:253: 
            # Derive module path from import comment. (0.026s)
            # Import comment works even with CRLF line endings. (0.058s)
            # Derive module path from location inside GOPATH.
            # 'go mod init' should succeed if modules are not explicitly disabled. (0.028s)
            # go mod init rejects a zero-length go.mod file (0.086s)
            # Module path from Godeps/Godeps.json overrides GOPATH. (0.064s)
            # Empty directory outside GOPATH fails. (0.024s)
            # Empty directory inside GOPATH/src uses location inside GOPATH. (0.008s)
            # In Plan 9, directories are automatically created in /n.
            # For example, /n/go.mod always exist, but it's a directory.
            # Test that we ignore directories when trying to find go.mod. (0.018s)
            > cd $WORK/gomoddir
            $WORK/gomoddir
            > ! go list .
            [stderr]
            go: cannot find main module, but found .git/config in /
                to create a module there, run:
                cd ../../../../../.. && go mod init
            [exit status 1]
            > stderr '^go: go.mod file not found in current directory or any parent directory; see ''go help modules''$'
            FAIL: testdata/script/mod_find.txt:52: no match for `(?m)^go: go.mod file not found in current directory or any parent directory; see 'go help modules'$` found in stderr
            
    --- FAIL: TestScript/mod_convert_dep (0.04s)
        script_test.go:253: 
            # We should not create a go.mod file unless the user ran 'go mod init' explicitly.
            # However, we should suggest 'go mod init' if we can find an alternate config file. (0.011s)
            # The command we suggested should succeed. (0.020s)
            # In Plan 9, directories are automatically created in /n.
            # For example, /n/Gopkg.lock always exists, but it's a directory.
            # Test that we ignore directories when trying to find alternate config files. (0.010s)
            > cd $WORK/gopkgdir/x
            $WORK/gopkgdir/x
            > ! go list .
            [stderr]
            go: cannot find main module, but found .git/config in /
                to create a module there, run:
                cd ../../../../../../.. && go mod init
            [exit status 1]
            > stderr '^go: go.mod file not found in current directory or any parent directory; see ''go help modules''$'
            FAIL: testdata/script/mod_convert_dep.txt:21: no match for `(?m)^go: go.mod file not found in current directory or any parent directory; see 'go help modules'$` found in stderr
            
FAIL
FAIL    cmd/go  190.854s
```

--- a/pkgs/applications/misc/gum/default.nix
+++ b/pkgs/applications/misc/gum/default.nix
@@ -30,7 +30,7 @@ buildGoModule rec {
 
   meta = with lib; {
     description = "Tasty Bubble Gum for your shell";
-    homepage = "https://github.com/charmbracelet/charm";
+    homepage = "https://github.com/charmbracelet/gum";
     changelog = "https://github.com/charmbracelet/gum/releases/tag/v${version}";
     license = licenses.mit;
     maintainers = with maintainers; [ maaslalani ];

--- a/pkgs/applications/misc/gum/default.nix
+++ b/pkgs/applications/misc/gum/default.nix
@@ -6,7 +6,7 @@ buildGoModule rec {
 
   src = fetchFromGitHub {
     owner = "charmbracelet";
-    repo = "gum";
+    repo = pname;
     rev = "v${version}";
     sha256 = "sha256-od0jJPfvczlru9hhO8ravGou6yYP91L2k37NYm2hD+k=";
   };

